src/mount_efs/__init__.py

```diff
@@ -1430,7 +1436,8 @@ def bootstrap_tls(
     state_file_dir=STATE_FILE_DIR,
     fallback_ip_address=None,
 ):
-    tls_port = choose_tls_port(config, options)
+    sock = choose_tls_port(state_file_dir, fs_id, mountpoint, config, options)
+    tls_port = sock.getsockname()[1]
```

Comment:
nit: Use a function for this `socket.getsockname()` such that in unit test you can directly use the func.

Reply:
Since we're always interested in both the socket and the port, I changed the function to return a tuple instead.